src/styles/index.scss

```diff
@@ -1,3 +1,107 @@
-body {
+// .scss
+$primary-color: #00acdc;
+$font-color: #060b35;
+$light-grey: #616070;
+$font-family: 'Roboto', sans-serif;
+
+html {
   margin: 0;
+  font-family: $font-family;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  margin: 0;
+  width: 200px;
+}
+
+.imac {
+  display: flex;
+  // padding-top: 32px;
+  justify-content: center;
+}
+
+.h2 {
+  color: $font-color;
+  text-align: left;
+  padding-top: 40px;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  padding-left: 16px;
+  padding-right: 16px;
+}
```

Comment:
Class names should be written in kebab-case. Please, rename the class '.h2' to follow this convention.